acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/lotus/FilecoinJsonRpcEndpoint.java

```diff
@@ -110,8 +118,8 @@ public static FilecoinSignedMessage walletSignMessage(
         .execute();
   }
 
-  public static String executeRawJsonRpcRequest(final String url, final String request)
-      throws IOException {
+  public @NotNull String executeRawJsonRpcRequest(final String request) throws IOException {
```

Comment:
nit: we don't use intellij annotations`@NotNull`

Reply:
done